aten/src/ATen/core/Tensor.h

```diff
@@ -295,10 +295,10 @@ class CAFFE2_API Tensor {
   Tensor & atan_();
```

Comment:
Turns out the style matters since in the autograd engine function signature generation, things would fail to build if you supplied `Generator*` somewhere and `Generator *` elsewhere.